pkg/webui/console/containers/gateway-status-panel/transmissions.js

```diff
@@ -33,17 +34,31 @@ import sharedMessages from '@ttn-lw/lib/shared-messages'
 import style from './gateway-status-panel.styl'
 
 const m = defineMessages({
-  statusLabel: 'Up and running for {days} days',
-  statusRecieved: 'Status received',
   noUplinks: 'No uplinks yet',
   noDownlinks: 'No downlinks yet',
   noStatus: 'No status',
   noDataYet: 'No data yet',
-  established: 'Established',
+  established: 'Established for {days}',
   notEstablished: 'Not established',
 })
 
-const Transmissions = ({ gatewayStats, isDisconnected, isUnavailable }) => {
+const calculateDaysPassed = timestamp => {
+  const oneDay = 24 * 60 * 60 * 1000 // Number of milliseconds in a day
+  const currentTime = new Date().getTime() // Current timestamp in milliseconds
+  const givenTime = new Date(timestamp).getTime() // Given timestamp in milliseconds
+
+  const timePassed = currentTime - givenTime
+
+  if (timePassed < oneDay) {
+    const hoursPassed = Math.floor(timePassed / (60 * 60 * 1000)) // Number of hours passed
+    return `${hoursPassed} hours`
+  }
+
+  const daysPassed = Math.floor(timePassed / oneDay) // Number of days passed
+  return `${daysPassed} days`
+}
```

Comment:
Why not `<DateTime.Relative />` ?

Reply:
That component uses this react-intl component: [https://formatjs.io/docs/react-intl/components/#formattedrelativetime](https://formatjs.io/docs/react-intl/components/#formattedrelativetime), which appends an "ago" word at the end (x days ago). In this case we want to say "for x days". I tried to use that component without the trailing "ago" word, but I couldn't remove it. So I had to find a workaround.

Reply:
Yeah, so the issue is that it should go on to say weeks, months, years ago if appropriate. Let's avoid recreating this altogether and use another wording then: `Connection started xx ago`.